src/main/kotlin/com/terraformation/backend/species/model/SpeciesModel.kt

```diff
@@ -45,10 +45,10 @@ data class SpeciesModel<ID : SpeciesId?>(
   companion object {
     fun of(
         record: Record,
-        ecosystemTypesMultiset: Field<Set<EcosystemType>>? = null,
-        growthFormsMultiset: Field<Set<GrowthForm>>? = null,
-        plantMaterialSourcingMethodsMultiset: Field<Set<PlantMaterialSourcingMethod>>? = null,
-        successionalGroupsMultiset: Field<Set<SuccessionalGroup>>? = null,
+        ecosystemTypesMultiset: Field<Set<EcosystemType>>,
+        growthFormsMultiset: Field<Set<GrowthForm>>,
+        plantMaterialSourcingMethodsMultiset: Field<Set<PlantMaterialSourcingMethod>>,
+        successionalGroupsMultiset: Field<Set<SuccessionalGroup>>,
```

Comment:
Unrelated?

Reply:
Yes, sorry, I am putting this back to how it was before my previous PR. I had merged it in  before I saw your comment, you are right, it is unrelated and should be put back to how it was before my last PR.